net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="netbios scanner"
+HOMEPAGE="http://nmbscan.gbarbier.org/"
+SRC_URI="http://nmbscan.gbarbier.org/down/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+
+RDEPEND="net-dns/bind-tools
```

Comment:
what about `DEPEND`?

Reply:
Actually, nmbscan is only a shell script, i think there is no need for a DEPEND?

Reply:
seems like, indeed, thanks for checking back.